src/wallet/walletdb.cpp

```diff
@@ -141,6 +158,18 @@ bool CWalletDB::WriteZKey(const libzcash::SproutPaymentAddress& addr, const libz
     // pair is: tuple_key("zkey", paymentaddress) --> secretkey
     return Write(std::make_pair(std::string("zkey"), addr), key, false);
 }
+bool CWalletDB::WriteSaplingZKey(const libzcash::SaplingIncomingViewingKey &ivk,
+                const libzcash::SaplingExtendedSpendingKey &key,
+                const CKeyMetadata &keyMeta,
+                const boost::optional<libzcash::SaplingPaymentAddress> &defaultAddr)
+{
+    nWalletDBUpdated++;
+    
+    if (!Write(std::make_pair(std::string("sapzkmeta"), ivk), keyMeta))
```

Comment:
Should this be "sapzkeymeta"? https://github.com/zcash/zcash/pull/3517/files#diff-e506682270036e4c8e0b8159a34b652dR715

If it should then this may indicate lack of test coverage.

Reply:
Yes, some of the naming conventions weren't very clear, so I missed this one. Will update for final pass.